e2e/fixtures/fdb_cluster.go

```diff
@@ -1016,8 +1039,23 @@ func (fdbCluster *FdbCluster) GetCustomParameters(
 	return fdbCluster.cluster.Spec.Processes[processClass].CustomParameters
 }
 
-// EnsurePodIsDeleted validates that a Pod is either not existing or is marked as deleted with a non-zero deletion timestamp.
-func (fdbCluster *FdbCluster) EnsurePodIsDeleted(podName string) {
+// CheckPodIsDeleted return (nil, true) if Pod no longer exists at the executed time point
+func (fdbCluster *FdbCluster) CheckPodIsDeleted(podName string) (bool, error) {
+	pod := &corev1.Pod{}
+	err := fdbCluster.getClient().
+		Get(ctx.TODO(), client.ObjectKey{Namespace: fdbCluster.Namespace(), Name: podName}, pod)
+
+	log.Println("error: ", err, "pod", pod.ObjectMeta)
+	if err != nil {
+		return kubeErrors.IsNotFound(err), err
```

Comment:
```suggestion
		if kubeErrors.IsNotFound(err) {
			return true, nil
		}

		return false, err
```

Otherwise the comment about the return value is wrong.